From 73a78fb9ed0fd2a6ad81b8b9ee17a35d57b58529 Mon Sep 17 00:00:00 2001
From: Juan Cruz Viotti <jv@jviotti.com>
Date: Fri, 17 Jan 2025 14:16:29 -0400
Subject: [PATCH] Fix `links` not adhering to its meta-schema given invalid
 `dependencies`

In Draft 4, `dependencies` is either a schema or an array of unique
strings.

See: https://json-schema.org/draft-04/draft-fge-json-schema-validation-00#rfc.section.5.4.5
See: https://www.learnjsonschema.com/draft4/validation/dependencies/
Signed-off-by: Juan Cruz Viotti <jv@jviotti.com>
---
 links.json | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/links.json b/links.json
index de272cc..5c0d30d 100644
--- a/links.json
+++ b/links.json
@@ -36,6 +36,6 @@
 	"required" : ["rel", "href"],
 	
 	"dependencies" : {
-		"enctype" : "method"
+		"enctype" : ["method"]
 	}
 }
\ No newline at end of file
-- 
2.48.0

